contracts/cross-chain/ZNSZChainPortal.sol

```diff
@@ -71,8 +71,12 @@ contract ZNSZChainPortal is UUPSUpgradeable, AAccessControlled, IZNSZChainPortal
         bytes32 parentHash,
         string calldata label,
         string calldata tokenURI
+    // TODO multi: figure this out !!!
 //      bool forceUpdateGlobalExitRoot  - do we need to pass this ???
     ) external override {
+        // TODO multi: can we optimize this somehow, so we don't have to do it every time ??
```

Comment:
Why are the other destination values set in the initializer but not this one? If we set this in the initializer we could be sure it's non-zero, as it also reverts when address(0) is given in the setter function below